.golangci.yaml

```diff
@@ -27,6 +27,9 @@ linters-settings:
       - octalLiteral
       - whyNoLint
       - unnamedResult
+    settings:
+      hugeParam:
+        sizeThreshold: 89
```

Comment:
I'm not sure where the warning is coming from, but maybe we should heed the warning and consider using pointers.

Reply:
It arose because the Network struct became heavy. I updated the code to use pointers which removes the relevant lint error